### PR TITLE
Feature/location

### PR DESCRIPTION
**Pull request by jpirazusta**
Ask user permission for getting current location and show it on the map.

gif

### CODE REVIEW

File: src/hooks/useLocation.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            ios: 'whenInUse',
+            android: {
+              detail: 'coarse',
+            },
+          });
+        }
+        if (granted) {
+          const latestLocation = await RNLocation.getLatestLocation({ timeout: 60000 });
+          const { latitude, longitude } = latestLocation;
+          setLocation({ latitude, longitude });
```
Comment (ManuViola77) [inline src/hooks/useLocation.js]: You can destructure in the first line, don't need to do it separately. 
`const { latitude, longitude } = await RNLocation.getLatestLocation({ timeout: 60000 });`

**Review by ManuViola77 — APPROVED** [review]

**Review by acollazomayer — APPROVED** [review]

File: src/constants/common.js [hunk 2]
```diff
@@ -1 +1,8 @@
 export const FACEBOOK_PERMISSIONS = ['email'];
+export const INITIAL_COORDS = {
```
Comment (peter7z) [inline src/constants/common.js]: I think you should'nt have initial coords. Maybe try to have them before showing the map?

**Review by peter7z — APPROVED** [review]
Nice!

File: src/hooks/useLocation.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  const [location, setLocation] = useState(INITIAL_COORDS);
+
+  useEffect(() => {
+    async function requestLocation() {
+      try {
+        let granted = await RNLocation.checkPermission({
+          ios: 'whenInUse',
+          android: {
+            detail: 'coarse',
+          },
```
Comment (elizabethlofredo) [inline src/hooks/useLocation.js]: You are reusing this object twice, what do you think about having this object on a constant "permissions" and use it then

File: src/hooks/useLocation.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+        if (!granted) {
+          granted = await RNLocation.requestPermission({
+            ios: 'whenInUse',
+            android: {
+              detail: 'coarse',
+            },
+          });
+        }
+        if (granted) {
+          const latestLocation = await RNLocation.getLatestLocation({ timeout: 60000 });
```
Comment (elizabethlofredo) [inline src/hooks/useLocation.js]: Also, you can add this 60000  into a constant :)

File: src/hooks/useLocation.js [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+        throw parseError(error);
+      }
+    }
+    requestLocation();
+  }, []);
+
+  return { location };
+};
+
+export default useLocation;
```
Comment (elizabethlofredo) [inline src/hooks/useLocation.js:47]: I like how you concentrate all this on one component :)

**Review by elizabethlofredo — APPROVED** [review]
Excellent job! I left some minor comments

**Pull request merged**